webpack/components/extensions/HostDetails/Tabs/PackagesTab.js

```diff
@@ -96,6 +226,42 @@ export const PackagesTab = () => {
     return newStatus;
   });
 
+  const actionButtons = (
+    <Split hasGutter>
+      <SplitItem>
+        <ActionList isIconList>
+          <ActionListItem>
+            <Dropdown
+              onSelect={onActionSelect}
+              toggle={
+                <DropdownToggle
+                  splitButtonItems={[
+                    <DropdownToggleAction key="action" aria-label="upgrade_actions" onClick={upgradeBulkViaRemoteExecution}>
```

Comment:
The Upgrade button should use Katello agent if that's what `defaultRemoteActionMethod` returns.  (This will align it with the behavior of the Apply button on the ErrataTab.)

Also, since the katello agent API takes package names, it should be disabled when in select all mode.

Reply:
But I think Katello agent should still work. I will ask Maria and see if I can get some feedback..